code/game/objects/items/cards_ids.dm

```diff
@@ -80,6 +80,7 @@
 	righthand_file = 'icons/mob/inhands/equipment/idcards_righthand.dmi'
 	item_flags = NO_MAT_REDEMPTION | NOBLUDGEON
 	var/prox_check = TRUE //If the emag requires you to be in range
+	var/uses = INFINITY // So playmin is still available

```

Comment:
What does this look like when you examine it ingame

Reply:
![billede](https://user-images.githubusercontent.com/15495375/60656732-42650600-9e50-11e9-97b1-72b2f42c016c.png)
woops...

Reply:
Make it not show uses if there are more than 50,000 uses left